ML_Project.py

```diff
@@ -123,9 +124,32 @@
 # Removed any rows missing data. Should only remove 2 rows missing embarked data.
 combined_df = combined_df.dropna()
 
+## Making embarkation data usable
+for dataset in combine:
+    dataset['Embarked'] = dataset['Embarked'].map( {'S': 0, 'C': 1, 'Q': 2} ).astype(int)
+
 # Confirm there are no more entries missing data
 #print(combined_df.isnull().sum())
 
+### Feature Engineering
+
+## Title
+for dataset in combine: # Perform this action for both the testing and training datasets
```

Comment:
I think there is some code missing here because there is nothing to pull the title out of each person's name.  As such this throws a `KeyError: 'Title'` because `Title` isn't a column of the dataframe